api/pkg/cli/installer/action.go

```diff
@@ -27,8 +28,14 @@ import (
 )
 
 const (
-	catalogLabel = "hub.tekton.dev/catalog"
-	versionLabel = "app.kubernetes.io/version"
+	versionLabel            = "app.kubernetes.io/version"
```

Comment:
Should we move this all labels to `hub.go` file, meaning trying to keep all constants at one place ?
wdyt ?

Reply:
Hmm, these labels are not used outside `action.go` file for now. Maybe we can move these constants if they got used somewhere else in the future?

Reply:
works 🙂